staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -185,10 +188,13 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	klog.V(3).Infof("Listing and watching %v from %s", r.expectedTypeName, r.name)
 	var resourceVersion string
 
-	// Explicitly set "0" as resource version - it's fine for the List()
-	// to be served from cache and potentially be delayed relative to
-	// etcd contents. Reflector framework will catch up via Watch() eventually.
-	options := metav1.ListOptions{ResourceVersion: "0"}
+	// Explicitly set resource version to have it list from cache for
+	// performance reasons.
+	// It's fine for the returned state to be stale (we will catch up via
+	// Watch() eventually), but can't set "0" to avoid going back in time
```

Comment:
suggest `It's fine for the returned state to be stale (we will catch up via Watch() eventually), but we need to be at least as new as the last resource version we synced to avoid going back in time.`

the "0" bit is confusing since we don't pass "0" here any more